app/pages/analyst/application-review.tsx

```diff
@@ -66,6 +69,11 @@ class ApplicationReview extends Component<Props> {
           applicationRevisionStatus={query.application.reviewRevisionStatus}
           applicationRowId={query.application.rowId}
         />
+        {overrideJustification && (
+          <ApplicationOverrideNotification
```

Comment:
The component below could return `null` if `overrideJustification` is falsy. This would make the code here simpler